backend/app/entity/shortlink.go

```diff
@@ -17,3 +17,28 @@ type ShortLink struct {
 	OpenGraphTags metatag.OpenGraph
 	TwitterTags   metatag.Twitter
 }
+
+// ShortLinkInput represents possible ShortLink attributes for a new short link or an updated short link.
+type ShortLinkInput struct {
+	LongLink    *string
+	CustomAlias *string
+	ExpireAt    *time.Time
+	CreatedAt   *time.Time
+	UpdatedAt   *time.Time
+}
+
+// GetLongLink fetches LongLink for ShortLinkInput, with empty string as the default value.
```

Comment:
```suggestion
// GetLongLink fetches LongLink for ShortLinkInput with default value.
```
We are passing `defaultVal` here, which not necessarily empty string.

Reply:
Nice catch. I wrote that docstring before I changed it. Thanks! 👍